.yarnrc.yml

```diff
@@ -1,5 +1,11 @@
+enableGlobalCache: true
+
+httpTimeout: 300000
+
 nodeLinker: node-modules
 
+npmRegistryServer: 'https://registry.npmjs.org/'
```

Comment:
The default registry, `registry.yarnpkg.com` is just a CNAME of the npm registry. Is there any tangible benefit to not using the default?

https://yarnpkg.com/getting-started/qa#why-registryyarnpkgcom-does-facebook-track-us

Reply:
would prefer the official hostname for the registry, one less variable for security.